src/components/container/Container.js

```diff
@@ -0,0 +1,32 @@
+import { fetchData } from '../../apiCalls';
+import React, { useEffect, useState } from 'react';
+import Show from '../show_card/ShowCard'
+import './Container.css'
+
+const Container = (props) => {
+  const [ isLoading, setIsLoading ] = useState(true);
+  const [ shows, setShows ] = useState([]);
+
+  //TO DO: includes conditional so that we can use this for our tracks and our years. 
+  useEffect(() => {
+    if (props.type === "shows") {
+      fetchData(`/years/${props.id}`).then(data => {
+        setShows(data.data);
+        setIsLoading(false);
+      }).catch(err => console.log(err))
+      //TO DO: when error component is made, render error component.
```

Comment:
We can use toastify to render error handling- it may already be in Nina's PR. we can go over it as a group.